src/tiles/list/tile-list-compact.ts

```diff
@@ -80,6 +82,22 @@ export class TileListCompact extends LitElement {
     return `${this.baseNavigationUrl}${basePath}${this.model.identifier}`;
   }
 
+  private get creator(): string | typeof nothing {
```

Comment:
This is a little duplicate ? Maybe putting a helper function on `model` might be cleaner, cause then you can do `this.model?.getDisplayedCreator(selectedCreatorFilter)`.

Reply:
Yeah, I'll try rethinking this a bit. It's tough because two of our three display modes use this new handling, while the third displays things another way entirely. I'm not sure I want the tile model to have to know about the display logic, but maybe this can be abstracted out somewhere cleaner.